{{cookiecutter.github_project_name}}/setup.py

```diff
@@ -33,7 +33,8 @@
 ]
 
 data_files_spec = [
-    ('share/jupyter/nbextensions/{{ cookiecutter.npm_package_name }}', '{{ cookiecutter.python_package_name }}/static', '*.*'),
+    ('share/jupyter/nbextensions/{{ cookiecutter.npm_package_name }}', '{{ cookiecutter.python_package_name }}/nbextension', '*.*'),
+    ('share/jupyter/labextensions/{{ cookiecutter.npm_package_name }}', '{{ cookiecutter.python_package_name }}/labextension', "*.*"),
```

Comment:
The pattern here needs to be `**/*.*` because the bundled files are placed in `labextension/static/`